tests/api/api/test_projects.py

```diff
@@ -956,29 +952,34 @@ def _create_resources_of_all_kinds(
                 )
 
     # Create several artifacts with several tags
-    artifact = {
-        "bla": "blabla",
-        "labels": labels,
+    artifact_template = {
+        "metadata": {"labels": labels},
+        "spec": {},
+        "kind": "artifact",
         "status": {"bla": "blabla"},
     }
     artifact_keys = ["artifact_key_1", "artifact_key_2", "artifact_key_3"]
-    artifact_uids = ["some_uid", "some_uid2", "some_uid3"]
+    artifact_trees = ["some_tree", "some_tree2", "some_tree3"]
```

Comment:
this test should pass with the uids as well
please test it both ways and add a comment to remove the uids in 1.8.0

Reply:
This test uses the sqldb directly to store artifact, where the uid is not used as the tree but as actual uid, thus it won't work we the `uid` argument.